### DISCUSSION TITLE
Can't create Codespaces

### DISCUSSION THREAD

**Discussion by tlockney** [community/community#4176] [Codespaces] [answered]
I haven't tried used Codespaces in a while, but came back today to try some experiments. I’ve been trying to create a new  instance today and it just won’t start. It just sits at the “Getting things ready…” spinner and never moves past that. This is on a test repository I created called [tlockney/workspace](https://github.com/tlockney/workspace), but there’s currently nothing in it other than a readme and a single shell script (I created the file through the Github UI directly, not through a Codespace or by cloning the repo locally). I also do not have a dotfiles repo, so nothing there that could be interfering with the startup. Any ideas?

While the Codespace never opens, if I go back and open my Codespaces view, I see the Codespace listed there, but when I try to open it, I get this message:

![Screenshot_2021-06-14_21-39-14](https://user-images.githubusercontent.com/8944/121993833-b675eb80-cd93-11eb-8cec-9f206788d9cf.png)

**Comment by derekbekoe** [comment, marked as answer]
@tlockney Thanks for reporting this. Codespaces had a temporary incident impacting a subset of customers that has since resolved. Let us know if you continue to see issues.
